glean-core/ios/Glean/GleanFfi.h

```diff
@@ -0,0 +1,430 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+typedef struct ExternError {
+    int32_t code;
+    char *message; // note: nullable
+} ExternError;
+typedef const char *FfiStr;
+typedef uint64_t TimerId;
+
+
+
+/* DO NOT MODIFY THIS MANUALLY! This file was generated using cbindgen.
```

Comment:
I wish this was at the beginning of the file, before `ExternError`. We can't really fix this, can't we?